src/applications/ezr/config/chapters/veteranInformation/contactInformation.js

```diff
@@ -27,7 +27,11 @@ export default {
           pattern: content['phone-number-error-message'],
         },
       },
-      email: emailUI(),
+      email: emailUI({
```

Comment:
Any chance there are tests somewhere that could have caught this or that could be updated to include this?

Reply:
Since these are web components, and not react-based... there isn't a way in mocha to get into the shadow DOM to test like you would want.